sdk/cosmos/azure-cosmos/test/test_query_async.py

```diff
@@ -12,9 +12,9 @@
 import azure.cosmos.aio._retry_utility_async as retry_utility
 import azure.cosmos.exceptions as exceptions
 import test_config
-from azure.cosmos import http_constants
+from azure.cosmos import http_constants, _endpoint_discovery_retry_policy
 from azure.cosmos._execution_context.query_execution_info import _PartitionedQueryExecutionInfo
-from azure.cosmos.aio import CosmosClient, DatabaseProxy, ContainerProxy
+from azure.cosmos.aio import CosmosClient, DatabaseProxy, ContainerProxy, _retry_utility_async
```

Comment:
`_retry_utility_async` has been imported already in line 12. You can remove this import and use `retry_utility` instead